python/linked_list.py

```diff
@@ -1,19 +1,68 @@
-class LinkList:
-  # write your __init__ method here that should store a 'head' value which the first Node in the LinkedList and a 'length' value which is the total number of Nodes in the LinkedList
+class LinkList():
+  def __init__(self, head=None):
+    self.head = head
+    self.length = 1
+    print(self.head)
+     # we're going to store the length of the Linked List here
 
   def add(self, data):
     # write your code to ADD an element to the Linked List
-    pass
+    self.length += 1
+    #for node in range(self.length-1)
+    node = self.head
+    while node.next != None:
+      node = node.next
+    node.next = data
 
   def remove(self, data):
-    # write your code to REMOVE an element from the Linked List
-    pass
+    self.length -=1
+    node = self.head
+    if node == data:
+      self.head = node.next
+    else:
+      while data != node:
+        previous = node
+        node = node.next
+      if node.next == None:
+        previous.next = None
+      else:
+        previous.next = node.next
+
+  def get(self, element):
+    node = self.head
+    for index in range(self.length-1):
+      if node == element:
+        return index
+      else:
+        node = node.next
 
-  def get(self, element_to_get):
-    # write you code to GET and return an element from the Linked List
-    pass
 
 # ----- Node ------
 class Node:
-  # store your DATA and NEXT values here
-  pass
+    def __init__(self, value):
+      self.value = value
+      self.next = None
+
+a = Node('a')
+b = Node('b')
+c = Node('c')
+d = Node('d')
+
+test = LinkList(a)
+test.add(b)
+test.add(c)
+test.add(d)
+
+
+print(test.head.value)
+print(test.head.next.value)
+print(test.head.next.next.value)
+print(test.head.next.next.next.value)
+
+print(test.get(c))
+print('-'*25)
```

Comment:
I like that you're testing out your LinkList, but it's best form to delete these before pull requesting, or at least putting them inside a if(__name__ == '__main__') statement so it doesn't run if someone wants to import your LinkList

Reply:
Also, this type of testing is perfect for a separate unittest file, and ideally that's where it should be. But I understand if you were running out of time and just wanted a quick and dirty way to test.